docs/extend/modular-resources/key-concepts.md

```diff
@@ -95,9 +95,11 @@ If you are [creating a custom module](/extend/modular-resources/create/) and [up
 - Your model triplet must be all-lowercase.
 - Your model triplet may only use alphanumeric (`a-z` and `0-9`), hyphen (`-`), and underscore (`_`) characters.
```

Comment:
note: namespace validations are more strict. We do not support `_` for them.

Reply:
we do now I believe